src/components/ui/Suggest.jsx

```diff
@@ -101,6 +102,31 @@ const Suggest = ({
     }
   };
 
+  useEffect(() => {
+    if (isMobile && dropdownVisible && window.visualViewport) {
```

Comment:
Note that `window.visualViewport` is not available by default on Firefox for desktop, including in the responsive view. 
(the flag `dom.visualviewport.enabled` can be enabled via "about:config").  
So it could make testing a bit confusing, especially for a newcomer. Do you think it could be documented somewhere?

Reply:
Good idea, I've added a link to the API doc.